payroll-processor-client/.prettierrc

```diff
@@ -1,11 +1,7 @@
 {
   "singleQuote": true,
   "trailingComma": "all",
-  "tabWidth": 2,
-  "useTabs": false,
-  "endOfLine": "auto",
   "semi": true,
-  "arrowParens": "avoid",
```

Comment:
Prettier recommends parens around single param arrow functions now and it's the default in Prettier 2.x